locales/th/php-inline.json

```diff
@@ -1,124 +1,124 @@
-    "missing_if": "ช่องนี้ต้องหายไปเมื่อ :other เป็น :value",
-    "missing_unless": "ต้องขาดฟิลด์นี้เว้นแต่ว่า :other จะเป็น :value",
-    "missing_with": "ช่องนี้ต้องหายไปเมื่อมี :values",
-    "missing_with_all": "ช่องนี้ต้องหายไปเมื่อมี :values",
-    "multiple_of": "ค่าจะต้องเป็นหลายของ :value",
-    "not_in": "ค่าที่เลือกไว้ใช้ไม่ได้หรือไม่ถูกต้อง",
-    "not_regex": "นี่รูปแบบไม่ถูกต้อง",
-    "numeric": "นี่ต้องเป็นตัวเลข",
-    "password.letters": "ช่องข้อมูลนี้ต้องประกอบด้วยตัวอักษรอย่างน้อยหนึ่งตัว",
-    "password.mixed": "ช่องข้อมูลนี้ต้องประกอบด้วยตัวอักษรพิมพ์ใหญ่อย่างน้อยหนึ่งตัว และตัวอักษรพิมพ์เล็กอย่างน้อยหนึ่งตัว",
-    "password.numbers": "ช่องข้อมูลนี้ต้องประกอบด้วยตัวเลขอย่างน้อยหนึ่งตัว",
-    "password.symbols": "ช่องข้อมูลนี้ต้องประกอบด้วยสัญลักษณ์อย่างน้อยหนึ่งตัว",
-    "password.uncompromised": "ค่าที่ป้อนถูกพบในข้อมูลที่รั่วไหล กรุณาใส่ค่าใหม่",
-    "present": "ช่องข้อมูลนี้ต้องเป็นปัจจุบัน",
-    "prohibited": "ช่องข้อมูลนี้เป็น prohibited.",
-    "prohibited_if": "ช่องข้อมูลนี้เป็น prohibited ตอนที่ :other คือ :value.",
-    "prohibited_unless": "ช่องข้อมูลนี้เป็น prohibited นอกจาก :other อยู่ใน :values.",
-    "prohibits": "ช่องข้อมูลนี้ห้าม :other ไม่ให้ถูกแสดง",
-    "regex": "นี่รูปแบบไม่ถูกต้อง",
-    "relatable": "ช่องข้อมูลนี้อาจจะไม่เกี่ยวข้องกับทรัพยากรนี้",
-    "required": "ช่องข้อมูลนี้เป็นต้องการ.",
-    "required_array_keys": "ช่องข้อมูลนี้ต้องประกอบด้วยรายการสำหรับ: :values.",
-    "required_if": "ช่องข้อมูลนี้ต้องถูกระบุเมื่อ :other มีค่าเท่ากับ :value.",
-    "required_if_accepted": "ช่องนี้จำเป็นเมื่อยอมรับ :other",
-    "required_unless": "ช่องข้อมูลนี้เป็นต้องการนอกจาก :other อยู่ใน :values.",
-    "required_with": "ช่องข้อมูลนี้เป็นต้องการตอนที่ :values เป็นของขวัญ",
-    "required_with_all": "ช่องข้อมูลนี้เป็นต้องการตอนที่ :values เป็นของขวัญ",
-    "required_without": "ช่องข้อมูลนี้เป็นต้องการตอนที่ :values ไม่ใช่ของขวัญ",
-    "required_without_all": "ช่องข้อมูลนี้เป็นต้องการตอนที่ไม่ใช่เรื่องของ :values เป็นของขวัญ",
-    "same": "ค่าของช่องข้อมูลนี้จะต้องตรงกับค่าใดค่าหนึ่งจาก :other.",
+    "attached": "ข้อมูลถูกแนบในฟิลด์นี้แล้ว",
+    "before": "ต้องระบุวันที่ก่อน :date.",
+    "before_or_equal": "ฟิลด์นี้ต้องเป็นวันที่ :date หรือก่อนหน้านั้น",
+    "between.array": "ข้อมูลส่วนนี้ต้องมีค่ามากกว่า :min และไม่เกิน :max รายการ",
+    "between.file": "ไฟล์นี้ต้องมีขนาดมากกว่า :min และไม่เกิน :max กิโลไบต์",
+    "between.numeric": "ข้อมูลต้องมีค่ามากกว่า :min และไม่เกิน :max",
+    "between.string": "ฟิลด์นี้ต้องมีความยาวตัวอักษรมากกว่า :min และไม่เกิน :max ตัวอักษร",
+    "boolean": "รองรับการระบุข้อมูล จริง(true) หรือ เท็จ(false) เท่านั้น",
+    "can": "ฟิลด์นี้มีค่าที่ไม่ได้รับอนุญาต",
+    "confirmed": "การยืนยันข้อมูลไม่ตรงกัน",
+    "country": "ข้อมูลที่ระบุไม่ใช่ประเทศที่ถูกต้อง",
+    "date": "ฟิลด์นี้ต้องระบุเป็นวันที่เท่านั้น",
+    "date_equals": "ข้อมูลที่ระบุต้องมีวันที่เท่ากับ :date",
+    "date_format": "ระบุข่อมูลไม่ตรงกับรูปแบบ :format ที่รองรับ",
```

Comment:
`ข่อมูล` should be `ข้อมูล`
So this one can be `"date_format": "ระบุข้อมูลไม่ตรงกับรูปแบบ :format ที่รองรับ",`